website/docs/docs/getting-started.mdx

```diff
@@ -60,6 +60,19 @@ We also have a Slack workspace where you can chat with other members and get gen
 
 And, if you need Pactflow specific technical support, email support@pactflow.io or for help with a PoC or plan options get in touch with sales.
 
+# Migrating from Pact Broker
+
+When you sign up to the SaaS Pactflow platform we create a new hosted database for you. However if you have previously used the open source Pact Broker for your contract testing needs we can help migrate the data so that nothing is lost. The process involves moving your existing database to the hosted environment associated with your Pactflow account, then updating it for full compatibility with Pactflow.
+This will require some scheduled down time for the service, including contract publishing and 'can-i-deploy' webhook.
+
+To get started with migrating from Pact Broker contact us at support@pactflow.io with the subject 'migrating from Pact Broker'.
+
+:::info
+Applies to SaaS (Hosted) Pactflow accounts only.
+
+Using On-Prem and want to migrate? Check out our on-prem migration guide [**here**](https://docs.pactflow.io/docs/on-premises/installation/migrating).
```

Comment:
Just to be SUPER clear, because users are dumb, how about `and ant to migrate to the SaaS platform?`

Reply:
and ant?

Reply:
oh and want

Reply:
AND ANT!!! AND ANT!!! 🐜

Reply:
don't forget to migrate your ants 🐜 🐜 🐜 🐜 🐜